_maps/shuttles/syndicate/syndicate_litieguai.dmm

```diff
@@ -1424,551 +1665,440 @@
-/turf/closed/wall/mineral/titanium/nodiagonal,
-/area/ship/maintenance/starboard)
-"IE" = (
-/obj/machinery/light/small/directional/east,
+"Hb" = (
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4,
 /obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2,
 /obj/structure/cable{
 	icon_state = "1-2"
 	},
-/turf/open/floor/plating,
-/area/ship/maintenance/port)
-"IG" = (
-/obj/effect/turf_decal/corner/opaque/red/full,
-/obj/machinery/atmospherics/components/unary/vent_scrubber/on/layer4{
+/obj/machinery/door/firedoor/border_only{
 	dir = 1
 	},
-/turf/open/floor/plasteel,
-/area/ship/cargo)
-"Ja" = (
-/obj/structure/chair{
-	dir = 4
-	},
-/obj/structure/extinguisher_cabinet/directional/west,
-/turf/open/floor/plasteel,
-/area/ship/cargo)
-"Je" = (
-/obj/machinery/power/terminal,
-/obj/structure/cable/yellow,
-/obj/item/wrench/crescent,
-/obj/item/radio/intercom/directional/west,
-/turf/open/floor/plating,
-/area/ship/maintenance/port)
-"Jn" = (
-/obj/machinery/power/shuttle/engine/electric{
-	dir = 1
+/obj/machinery/door/firedoor/border_only,
+/obj/machinery/door/airlock/maintenance_hatch{
+	name = "Port Engines"
 	},
-/obj/structure/cable,
 /turf/open/floor/plating,
 /area/ship/maintenance/port)
-"JA" = (
-/obj/effect/spawner/structure/window/shuttle,
-/obj/machinery/door/poddoor/preopen{
-	dir = 4;
-	id = "traumawindows";
-	name = "Window Blast Door"
-	},
-/turf/open/floor/plating,
-/area/ship/medical)
-"Kc" = (
-/obj/structure/sign/departments/medbay/alt,
+"Hk" = (
+/obj/effect/turf_decal/industrial/outline/red,
+/obj/structure/crate_shelf,
+/obj/machinery/light/small/directional/west,
+/turf/open/floor/plasteel/tech,
+/area/ship/storage)
+"Hl" = (
+/obj/structure/sign/poster/contraband/random,
 /turf/closed/wall/mineral/titanium/nodiagonal,
 /area/ship/crew)
-"Kh" = (
-/obj/machinery/atmospherics/components/unary/vent_pump/on/layer2{
-	dir = 1
-	},
-/obj/machinery/airalarm/directional/west,
-/obj/machinery/power/terminal,
-/obj/structure/cable,
-/obj/effect/turf_decal/industrial/warning,
-/turf/open/floor/plating,
-/area/ship/maintenance/port)
-"Ki" = (
-/obj/effect/landmark/start/paramedic,
-/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2,
-/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4,
-/obj/machinery/firealarm/directional/west,
-/obj/machinery/light_switch{
-	dir = 4;
-	pixel_x = -21;
-	pixel_y = -10
-	},
-/turf/open/floor/plasteel,
-/area/ship/crew)
-"Ko" = (
-/obj/machinery/light/directional/east,
+"HE" = (
+/obj/structure/rack,
 /obj/effect/turf_decal/industrial/outline/red,
-/obj/machinery/suit_storage_unit/inherit,
-/obj/item/clothing/suit/space/hardsuit/medical,
-/obj/item/clothing/mask/gas/sechailer,
-/obj/item/tank/internals/emergency_oxygen/engi,
+/obj/item/circuitboard/machine/ore_redemption,
+/obj/item/stack/sheet/glass/fifty,
+/obj/item/stack/sheet/metal/fifty,
+/obj/item/circuitboard/machine/rdserver,
+/obj/item/circuitboard/computer/rdconsole,
```

Comment:
We can just make the RnD machines pre-built, these boards are from way back when RnD machines refused to link when ships spawned in.